### PR TITLE
feat: nip05 on profile page

### PR DESCRIPTION
**Pull request by verbiricha**
Noticed cash.app doesn't set CORS so those don't show as verified but in general the nostr.json files are CORS-ready

Screenshot 2023-01-09 at 19 42 31

### CODE REVIEW

File: src/element/Nip05.js [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+
+import './Nip05.css'
+
+const Nip05 = ({ nip05, pubkey }) => {
+    const [nip05pubkey, setNip05pubkey] = useState()
+    const isVerified = nip05pubkey === pubkey
+    const [nick, domain] = nip05.split('@')
+
+    useEffect(() => {
+        fetch(`https://${domain}/.well-known/nostr.json`)
```
Comment (v0l) [inline src/element/Nip05.js:14]: You should include the param `?name=${nick}` some services will require this

File: src/element/Nip05.js [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+          .catch((err) => {
+            console.error("Couldn't verifiy nip05")
+          })
+    }, [nip05, nick, domain])
+
+    return (
+       <div className="flex nip05">
+         <div className="nick">{nick}</div>
+         <div className="domain">@{domain}</div>
+         {isVerified && (
```
Comment (v0l) [inline src/element/Nip05.js:30]: If its not verified then we should show error right?

File: src/element/Nip05.js (hunk 1 above)
Comment (v0l) [reply src/element/Nip05.js:14]: https://github.com/nostr-protocol/nips/blob/master/05.md

Comment (verbiricha) [reply src/element/Nip05.js]: Good point, done.

File: src/element/Nip05.js (hunk 2 above)
Comment (verbiricha) [reply src/element/Nip05.js:37]: Screenshot 2023-01-10 at 07 19 08

File: src/element/Nip05.js (hunk 1 above)
Comment (verbiricha) [reply src/element/Nip05.js]: Also treating `_` as the default user and following spec recommendation for display:
Screenshot 2023-01-10 at 07 18 49
Screenshot 2023-01-10 at 07 18 44

**Review by v0l — APPROVED** [review]

**Pull request merged**